README.md

```diff
@@ -67,6 +67,7 @@ recommend to set at least these values:
 | `nginx.args`                           | Override the default arguments to run in the container                                     | `[]`              |
 | `duckling.command`                     | Override the default command to run in the container                                       | `[]`              |
 | `duckling.args`                        | Override the default arguments to run in the container                                     | `[]`              |
+| `global.progressDeadlineSeconds`       | Specifies the number of seconds you want to wait for your Deployment to progress before the system reports back that the Deployment has failed progressing | `600` |
```

Comment:
We somewhat only listed the most important ones here. Imo no need to list this one here as it should only be changed rarely, wdyt?

Reply:
I've put the variable here as it a good practice to have all parameters in the README file, even if they don't change very often.

Reply:
I always disliked it because it means maintaining two locations. Up to you, but then we should create an issue for the remaining entries to be consistent.